sympy/geometry/ellipse.py

```diff
@@ -1343,14 +1343,14 @@ class Circle(Ellipse):
 
     >>> # a circle object is returned
     >>> from sympy.abc import x, y, a, b
-    >>> Circle(equation = x ** 2 + y ** 2 - 25)
+    >>> Circle(equation = x ** 2 + y ** 2 - 25, x='x', y='y')
```

Comment:
You can set the default values of `x` and `y` to be 'x' and 'y', in a similar fashion as done for `equation`, so that the user doesn't have to enter that everytime.

Reply:
@ArifAhmed1995  Yes, I was working on that earlier too. Now it's incorporated in the code. Please take a look.